appveyor.yml

```diff
@@ -0,0 +1,16 @@
+version: 'alpha4.{build}'
+pull_requests:
+  do_not_increment_build_number: true
+image: WMF 5
```

Comment:
This line is only needed if you need the latest PowerShell. Otherwise you can drop it.